### ISSUE TITLE
[Render thread/ERROR]: Mod head_in_the_clouds provides a broken implementation of ModMenuApi

### ISSUE THREAD

**Issue by Dronatar** [DaFuqs/HeadInTheClouds#9]
MC `1.21.3`
Fabric Loader `0.16.9`
Mods:
> Head in the Clouds `1.2.0`
Mod Menu `12.0.0-beta.1`
Fabric API `0.109.0+1.21.3`
Cloth Config API `16.0.141+fabric`
Text Placeholder API `2.5.1+1.21.3`

Logs:
```
[Render thread/ERROR]: Mod head_in_the_clouds provides a broken implementation of ModMenuApi
net.fabricmc.loader.api.EntrypointException: Exception while loading entries for entrypoint 'modmenu' provided by 'head_in_the_clouds'
	at net.fabricmc.loader.impl.entrypoint.EntrypointContainerImpl.getEntrypoint(EntrypointContainerImpl.java:56) ~[fabric-loader-0.16.9.jar:?]
	at knot/com.terraformersmc.modmenu.ModMenu.lambda$onInitializeClient$0(ModMenu.java:104) ~[modmenu-12.0.0-beta.1.jar:?]
	at java.base/java.util.ArrayList.forEach(Unknown Source) ~[?:?]
	at knot/com.terraformersmc.modmenu.ModMenu.onInitializeClient(ModMenu.java:99) ~[modmenu-12.0.0-beta.1.jar:?]
	at net.fabricmc.loader.impl.FabricLoaderImpl.invokeEntrypoints(FabricLoaderImpl.java:399) [fabric-loader-0.16.9.jar:?]
	at net.fabricmc.loader.impl.game.minecraft.Hooks.startClient(Hooks.java:53) [fabric-loader-0.16.9.jar:?]
	at knot/net.minecraft.class_310.<init>(class_310.java:482) [client-intermediary.jar:?]
	at knot/net.minecraft.client.main.Main.main(Main.java:248) [client-intermediary.jar:?]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:480) [fabric-loader-0.16.9.jar:?]
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74) [fabric-loader-0.16.9.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23) [fabric-loader-0.16.9.jar:?]
Caused by: net.fabricmc.loader.api.LanguageAdapterException: java.lang.ClassNotFoundException: de.dafuqs.head_in_the_clouds.config.ModMenuConfig
	at net.fabricmc.loader.impl.util.DefaultLanguageAdapter.create(DefaultLanguageAdapter.java:52) ~[fabric-loader-0.16.9.jar:?]
	at net.fabricmc.loader.impl.entrypoint.EntrypointStorage$NewEntry.getOrCreate(EntrypointStorage.java:124) ~[fabric-loader-0.16.9.jar:?]
	at net.fabricmc.loader.impl.entrypoint.EntrypointContainerImpl.getEntrypoint(EntrypointContainerImpl.java:53) ~[fabric-loader-0.16.9.jar:?]
	... 10 more
Caused by: java.lang.ClassNotFoundException: de.dafuqs.head_in_the_clouds.config.ModMenuConfig
	at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(Unknown Source) ~[?:?]
	at java.base/java.lang.ClassLoader.loadClass(Unknown Source) ~[?:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.loadClass(KnotClassDelegate.java:226) ~[fabric-loader-0.16.9.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:119) ~[fabric-loader-0.16.9.jar:?]
	at java.base/java.lang.ClassLoader.loadClass(Unknown Source) ~[?:?]
	at java.base/java.lang.Class.forName0(Native Method) ~[?:?]
	at java.base/java.lang.Class.forName(Unknown Source) ~[?:?]
	at java.base/java.lang.Class.forName(Unknown Source) ~[?:?]
	at net.fabricmc.loader.impl.util.DefaultLanguageAdapter.create(DefaultLanguageAdapter.java:50) ~[fabric-loader-0.16.9.jar:?]
	at net.fabricmc.loader.impl.entrypoint.EntrypointStorage$NewEntry.getOrCreate(EntrypointStorage.java:124) ~[fabric-loader-0.16.9.jar:?]
	at net.fabricmc.loader.impl.entrypoint.EntrypointContainerImpl.getEntrypoint(EntrypointContainerImpl.java:53) ~[fabric-loader-0.16.9.jar:?]
	... 10 more

**Comment by DaFuqs** [comment]
Good catch, thank you!

**Comment by DaFuqs** [comment]
Fixed and released in 1.2.1!
